### ISSUE TITLE
[BUG] ‘gcc’不是内部或外部命令/LLM对话只能持续一轮

### ISSUE THREAD

**Issue by mlwei1** [chatchat-space/Langchain-Chatchat#542]
No compiled kernel found.
Compiling kernels : C:\Users\scx56\.cache\huggingface\modules\transformers_modules\chatglm-6b-int4\quantization_kernels_parallel.c
Compiling gcc -O3 -fPIC -pthread -fopenmp -std=c99 C:\Users\scx56\.cache\huggingface\modules\transformers_modules\chatglm-6b-int4\quantization_kernels_parallel.c -shared -o C:\Users\scx56\.cache\huggingface\modules\transformers_modules\chatglm-6b-int4\quantization_kernels_parallel.so
'gcc' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
Compile default cpu kernel failed, using default cpu kernel code.
Compiling gcc -O3 -fPIC -std=c99 C:\Users\scx56\.cache\huggingface\modules\transformers_modules\chatglm-6b-int4\quantization_kernels.c -shared -o C:\Users\scx56\.cache\huggingface\modules\transformers_modules\chatglm-6b-int4\quantization_kernels.so
'gcc' 不是内部或外部命令，也不是可运行的程序
或批处理文件。
Compile default cpu kernel failed.
Failed to load kernel.
Cannot load cpu kernel, don't use quantized model on cpu.
Using quantization cache
Applying quantization to glm layers
Loaded the model in 11.34 seconds.
INFO  2023-06-03 22:25:27,699-1d: Load pretrained SentenceTransformer: GanymedeNil/text2vec-large-chinese
WARNING 2023-06-03 22:25:30,585-1d: No sentence-transformers model found with name C:\Users\scx56/.cache\torch\sentence_transformers\GanymedeNil_text2vec-large-chinese. Creating a new one with MEAN pooling.
WARNING 2023-06-03 22:25:45,009-1d: The dtype of attention mask (torch.int64) is not bool
{'answer': '你好👋！我是人工智能助手 ChatGLM-6B，很高兴见到你，欢迎问我任何问题。'}
INFO  2023-06-03 22:25:59,769-1d: 模型已成功加载，可以开始对话，或从右侧选择模式后开始对话
Running on local URL:  http://localhost:7860
Running on public URL: https://df31bacf034d5a2f39.gradio.live

**复现问题的步骤 / Steps to Reproduce**
启动webui.py,载入过程中出现GCC错误，不影响正常启动，但是webui启动后LLM只能有一轮对话，设置的是3轮。知识库回答正常

**环境信息 / Environment Information**
- langchain-ChatGLM 版本/commit 号：(例如：v1.0.0 或 commit 123456) / langchain-ChatGLM version/commit number: (e.g., v1.0.0 or commit 123456)
- 是否使用 Docker 部署（是/否）：否
- 使用的模型（ChatGLM-6B / ClueAI/ChatYuan-large-v2 等）：ChatGLM-6B-int4
- 使用的 Embedding 模型（GanymedeNil/text2vec-large-chinese 等）：GanymedeNil/text2vec-large-chinese 
- 操作系统及版本 / Operating system and version:win10
- Python 版本 / Python version:3.8
- 其他相关环境信息 / Other relevant environment information:
